src/tab.ts

```diff
@@ -81,14 +109,6 @@ export default class Tab extends LitElement {
   @property({ reflect: true })
   override readonly role = 'tab';
 
-  privateSelect() {
```

Comment:
Removed `privateSelect()` in favor of Tab Group setting `selected` directly instead.